dynamiqs/qarrays/utils.py

```diff
@@ -288,6 +287,8 @@ def to_qutip(x: QArrayLike, dims: tuple[int, ...] | None = None) -> Qobj | list[
          [0. 0. 0. 0. 1. 0.]
          [0. 0. 0. 0. 0. 1.]]
     """  # noqa: E501
+    from .._checks import check_shape
+
```

Comment:
For some reason, removing the dark namespace required this to avoid a circular import. The fact that there is a circular import makes sense, but I'm not sure why the dark namespace made it work.